libkbfs/keybase_daemon.go

```diff
@@ -119,3 +119,15 @@ func (k keybaseDaemon) NewCrypto(config Config, params InitParams, ctx Context,
 	}
 	return crypto, nil
 }
+
+func (k keybaseDaemon) NewChat(
```

Comment:
[My PR](https://github.com/keybase/kbfs/pull/1604) changes the init code a little bit. If we end up deciding merging that, this could probably go into "AdditionalProtocolCreaters".

Reply:
Ah nvm I confused myself. This is a client not handler.